api_docs/changelog.md

```diff
@@ -20,6 +20,14 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 268**
+
+* [`POST /register`](/api/register-queue), [`PATCH /settings`](/api/update-settings),
+  [`PATCH /realm/user_settings_defaults`](/api/update-realm-user-settings-defaults):
+  Added new user setting `web_channel_default_view`. Clients may use this to
+  determine which narrow to show on clicking a channel link in a UI component like
```

Comment:
"which view", not "which narrow"; we're phasing out the "narrow" term, and also we should keep in mind that some things, like a "channel inbox", would not even be "narrows/message views" technically.

Reply:
Thanks, noted and changed that here.